jujugui/static/gui/src/app/app.js

```diff
@@ -1828,9 +1833,15 @@ YUI.add('juju-gui', function(Y) {
             root: null
           });
         } else {
-          // If no model was found then put the user in disconnected mode.
+          // If no model was found then navigate to the user profile.
+          const msg = `no such charm, bundle or model: u/${modelPath}`;
+          // TODO frankban: here we should put a notification, but we can't as
```

Comment:
This has me a little concerned, I'll add it to my list of things to investigate.